contributing_to_docs/term_glossary.adoc

```diff
@@ -423,7 +423,7 @@ When referencing as a prerequisite for a procedure module, use the following
 construction: Install the OpenShift CLI (`oc`).
 
 ''''
-=== OpenShift control plane (also known as master)
```

Comment:
@Amrita42 This update LGTM. However, I wonder whether it would be worthwhile to call out in the description below that it was previously known as "master" (and sometimes still is in the code), but that when possible, "master" should be replaced with "control plane"? wdyt?

Reply:
@bergerhoffer I think it is a good idea to mention that, so I have added a sentence to that effect. ptal